testsuite/standalone/src/test/java/org/wildfly/extras/creaper/core/online/operations/admin/Reload500TimesHttpTest.java

```diff
@@ -0,0 +1,100 @@
+package org.wildfly.extras.creaper.core.online.operations.admin;
+
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.auth.PropertiesFileAuth;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+import org.wildfly.extras.creaper.core.online.ManagementProtocol;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.test.SlowTests;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@Category(SlowTests.class)
+@RunWith(Arquillian.class)
+public class Reload500TimesHttpTest extends Reload500TimesTest {
```

Comment:
This doesn't really have to extend `Reload500TimesTest`, no? The inheritance doesn't give you anything. You could possibly refactor `Reload500TimesTest` so that the client is created in a `@Before` method. Then, it would make sense to inherit from that and only override the `@Before` method, inheriting both the `@Test` methods for free.